plugins/win-mf/libmfcapture/source/MediaFoundationSourceInput.h

```diff
@@ -0,0 +1,147 @@
+#pragma once
+
+#include <obs-module.h>
+#include <obs.hpp>
+#include <util/dstr.hpp>
+#include <util/platform.h>
+#include <util/threading.h>
+#include <util/util.hpp>
+#include <util/windows/WinHandle.hpp>
+
+#include <algorithm>
+#include <limits>
+#include <set>
+#include <string>
+#include <vector>
+
+#include <dxcore.h>
+#include <mfapi.h>
+#include <objbase.h>
+#include <winrt/base.h>
+#include <initguid.h>
+
+#include "mfcapture.hpp"
+#include "win-mf.hpp"
+
+extern DWORD CALLBACK MediaFoundationSourceThread(LPVOID ptr);
+
+struct MediaFoundationSourceInput {
+	obs_source_t *source;
+	CAPTURE_DEVICE_HANDLE _mfcaptureDevice = nullptr;
```

Comment:
"Private" variables should have trailing underscores in C++ to avoid clobbering reserved names.

Given that this is a `struct` all members are "public" any way - what is the reason to use a naming scheme that implies "private" membership here?